proto/streamlit/proto/Radio.proto

```diff
@@ -19,6 +19,6 @@ syntax = "proto3";
 message Radio {
   string id = 1;
   string label = 2;
-  int32 default = 3;
+  int32 value = 3;
```

Comment:
Like Button.proto comment: To change the protobuf, choose a new numerical value for "value" and explicitly deprecate the word "default" as well as its number.